async.go

```diff
@@ -306,12 +305,12 @@ func (c *Async) killGoroutines() {
 	c.wg.Wait()
 	_ = c.conn.SetDeadline(emptyTime)
 	close(c.closeCh)
-	c.Logger().Error().Msg("error channel closed, goroutines killed")
+	c.Logger().Debug().Msg("error channel closed, goroutines killed")
```

Comment:
why are we changing these to debug lines?

Reply:
Because it's not actually an error when the error channel is closed, it's necessary whenever we close a frisbee.Conn